src/run-cache.test.ts

```
@@ -1103,4 +1103,174 @@ describe("RunCache", () => {
+      // Check that no entries were evicted despite exceeding max size
+      await expect(RunCache.get("key1")).resolves.toBe("value1");
+      await expect(RunCache.get("key2")).resolves.toBe("value2");
+      await expect(RunCache.get("key3")).resolves.toBe("value3");
+      await expect(RunCache.get("key4")).resolves.toBe("value4");
+      await expect(RunCache.get("key5")).resolves.toBe("value5");
+    });
+
+    it("should correctly update configuration", async () => {
+      // Check initial configuration
+      expect(RunCache.getConfig()).toEqual({
+        maxSize: Infinity,
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)